### ISSUE TITLE
Dependency on the AtlasRep package

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/HeLP#7]
The following is displayed if HeLP is loaded with OnlyNeeded option:
```
%%% Loading help 3.4 , only needed
Syntax warning: Unbound global variable in /circa/scratch/gap-jenkins/workspac\
e/GAP-pkg-update-stable-quicktest/GAPCOPTS/64build/GAPGMP/gmp/GAPTARGET/packag\
esload/label/kovacs/GAP-pkg-update-stable-snapshot/pkg/HeLP-3.4/lib/HeLP.gi:12\
67
  G := AtlasGroup(Identifier(C));
       ^^^^^^^^^^
```
Either AtlasRep should be made a needed package, or some code should be loaded optionally.

**Comment by margollo** [comment]
Solved in new release 3.5. AtlasRep has been added to dependencies.

**Comment by alex-konovalov** [comment]
Reopened this - can't see changes in the master branch.

**Comment by alex-konovalov** [comment]
Now properly closed in 28fda0373be66df24ab0038105feb69709803dea
